python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -121,6 +121,49 @@ def _make_ref_covariance_matrix(
     return cov
 
 
+def _nCoeffsFromDegree(degree):
+    """Get the number of coefficients for a polynomial of a certain degree with
+    two variables.
+
+    This uses the general formula that the number of coefficients for a
+    polynomial of degree d with n variables is (n + d) choose d, where in this
+    case n is fixed to 2.
+
+    Parameters
+    ----------
+    degree : `int`
+        Degree of the polynomial in question.
+
+    Returns
+    -------
+    nCoeffs : `int`
+        Number of coefficients for the polynomial in question.
+    """
+    nCoeffs = int(((2 * degree + 3) ** 2 - 1) / 8)
```

Comment:
This formula wasn't obvious but makes sense now after reading the docstring. Why not just `(degree+2)*(degree+1)/2` though? It doesn't seem like the simplest form of the quantity you want to compute.

Reply:
Yeah, I don't remember why I didn't simplify this when I first implemented it. I think my idea was to preserve the structure of the formula, but I don't think it's necessary.